test-variables.yml.example

```diff
@@ -19,3 +19,15 @@ OCP_ADMIN_USER:
 S3:
   AWS_ACCESS_KEY_ID: ID-value
   AWS_SECRET_ACCESS_KEY: Secret-Key
+ANACONDA_CE:
+  ACTIVATION_KEY: license-key
+SSO:
+  USERNAME: rh-sso-account
+  PASSWORD: rh-sso-account-pw
+RHOSAK_CONFIG_TEST:
```

Comment:
Is `_CONFIG_TEST `needed?    Otherwise LGTM

Reply:
I thought it would have been more self-explanatory while calling it in the code. But yeah it's not extremely important

Reply:
if you want I can change it, but IMO the effort is not worth now to do it